addons/docs/src/frameworks/vue3/extractArgTypes.ts

```diff
@@ -33,6 +33,13 @@ export const extractArgTypes: ArgTypesExtractor = (component) => {
           category: section,
         },
       };
+
+      // split props, slots etc which could clash on name
+      if (results.section === undefined) {
```

Comment:
This will be `undefined` every time, you have to check for the key: `results[section]`

Reply:
Good catch, building storybook was a bit slow for me.